src/components/LinearGradient/ClearButton.js

```diff
@@ -9,12 +9,12 @@ import { View, TouchableOpacity, StyleSheet, TextInput, Text } from "react-nativ
  * Optional props:
  *      customStyles
  *      customTextStyles
+ *      unlockButton -> home screen active button
  *      
  *      
  */
 
```

Comment:
use proptypes instead of this... 
https://hackernoon.com/are-you-using-proptypes-in-react-native-6067e2e5b526